cmd/kusk/cmd/generate.go

```diff
@@ -119,7 +118,9 @@ var generateCmd = &cobra.Command{
 
 	This will fetch the OpenAPI document from the provided URL and generate a Kusk Gateway API resource
 	`,
-	Run: func(cmd *cobra.Command, args []string) {
+	SilenceUsage:  true,
+	SilenceErrors: true,
```

Comment:
We don't need to have this on each command. If we want all commands to SilenceErrors and SilenceUsage, we can just add it to the Root cmd